conda_forge_tick/auto_tick.xsh

```diff
@@ -88,20 +126,24 @@ def main():
                 rerender = (gx.nodes[node].get('smithy_version') != smithy_version or
                             gx.nodes[node].get('pinning_version') != pinning_version or
                             migrator.rerender)
-                run(attrs=attrs, migrator=migrator, gh=gh, rerender=rerender, protocol='https',
-                    hash_type=attrs.get('hash_type', 'sha256'))
-                # TODO: capture pinning here too!
-                gx.nodes[node].update({'PRed': attrs['new_version'],
-                                       'smithy_version': smithy_version,
+                migrator_hash, pr_json = run(attrs=attrs, migrator=migrator, gh=gh,
+                                    rerender=rerender, protocol='https',
+                                    hash_type=attrs.get('hash_type', 'sha256'))
+
+                # TODO: convert this list to set and use something other
+                # than dict for the hash (namedtuple?)
+                gx.nodes[node].setdefault('PRed', []).append(migrator_hash)
+                # Stash the pr json data so we can access it later
+                gx.nodes[node].setdefault('PRed_json', {}).update(
```

Comment:
This will allow us to access the actual `github3.py` PR objects for the bot PRs (eg close them, ping maintainers, check status)